__tests__/test.js

```diff
@@ -45,3 +105,79 @@ describe('getOrThrow', () => {
         expect(getOrThrow('TIMEZONE')).to.equal(process.env.TIMEZONE);
     });
 });
+
+describe('createIdObject', () => {
+    it('should create a object id', () => {
+        const returnObject = createIdObject(
+            ['mockedUser', 'mockedUser@user.com'],
+            mockedContributed
+        );
+
+        expect(returnObject.alternateId).to.equal('mockedUser@user.com');
+        expect(returnObject.github).to.equal('mockedUser');
+
+        const contributionsObject = [
+            { id: '1', type: 'IssueCommentEvent' },
```

Comment:
Rather than duplicating the data here, could this be: `{id: mockedContributed[0].id, type: mockedContributed[0].type },`?

That would also make it a bit more clear why the loop below is relying on the index position within the array.